pkg/aspectsupport/accessLogger/config/config.proto

```diff
@@ -0,0 +1,45 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package pkg.adaptersupport.accessLogger.config;
+
+option go_package="config";
+
+// Configures an individual logger aspect implementation.
+//
+// Example usage:
+// kind: istio/accessLogger
+// params:
+//    log_name: "logs/access_log"
+//    attributes: "source.ip","host.URL","
+message Params {
+    // Provides a way to identify a collection of related log entries.
+    string log_name = 1;
+
+    // A list of names of attributes that will be used to generate
+    // access log entries.
+    repeated string attributes = 2;
+
+    // This will format string will be applied to the set of
+    // attributes specified to generate a textual access log entry.
+    //
+    // If this is not specified, the mixer will attempt to use a
+    // default format that closely aligns with the Common Log Format.
+    //
+    // Default:
+    // `{{or (.source_ip) "-"}} - {{or (.source_user) "-"}} [{{.timestamp.Format "02/Jan/2006:15:04:05 -0700"}}] "{{or (.method) "-"}} {{or (.api_resource) "-"}} {{or (.protocol) "-"}}" {{or (.status_code) "-"}} {{or (.response_size) "-"}}`
```

Comment:
go template expression language is not the easiest to read, but for now it gets the point across.
Once we have an expression evaluator, we can use the same language.